doc/README-2.0

```diff
@@ -162,7 +158,7 @@ Usage: pkitool [options...] [common-name]
 Options:
   --batch    : batch mode (default)
   --keysize  : Set keysize
-      size   : size (default=1024)
+      size   : size (default=4096)
```

Comment:
I think 4096 is too much, in particular for DHE. Both parameter generation and connection setup will get pretty slow.  I can see three ways out of this:
 * Use 2048 or 3072 as the default key size
 * Split default key sizes for DHE and RSA (e.g. 3072 / 4096)
 * Use the IETF ffdh parameters, instead of generating our own